pkgs/applications/virtualization/open-vm-tools/default.nix

```
@@ -23,6 +23,13 @@ stdenv.mkDerivation rec {
   buildInputs = [ fuse glib icu libdnet libmspack libtirpc openssl pam procps rpcsvc-proto xercesc ]
       ++ lib.optionals withX [ gdk-pixbuf-xlib gtk3 gtkmm3 libX11 libXext libXinerama libXi libXrender libXrandr libXtst ];
 
+  patches = [
+    # Fix building with glib 2.68. Remove after next release.
+    # The patch drops AUTHORS changes due to conflicts, and is further altered for sourceRoot
+    # https://github.com/vmware/open-vm-tools/pull/505
+    ./glib-2.68.patch
```

Oh! I think we can work around these issues without having to include the patch in-tree!

```suggestion
    (fetchpatch {
      url = "https://github.com/vmware/open-vm-tools/commit/82931a1bcb39d5132910c7fb2ddc086c51d06662.patch";
      stripLen = "1";
      excludes = [ "AUTHORS" ];
      sha256 = "0yz5hnngr5vd4416hvmh8734a9vxa18d2xd37kl7if0p9vik6zlg";
    })
```

---

Thanks! I've force pushed - I should have looked whether those changes are possible inside fetchpatch to start with :sweat_smile: